common/share widget/reports/ally-1-en.html

```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Accessibility assessment #1 - Gadget de partage",
+	"language": "en",
+	"description": "Partial. Evaluation of new Success Criteria at Level A and Level AA of WCAG 2.1 and WCAG 2.2",
+	"tag": "share",
+	"parentdir": "share",
+	"parentPage": "Gadget de partage",
+	"parentPageURL": "share",
+	"altLangPage": "a11y-1-fr.html",
```

Comment:
The alternate language page is
```suggestion
	"altLangPage": "ally-1-fr.html",
```